Fixes the following errors:

> In file included from ../../third_party/openscreen/src/util/crypto/random_bytes.cc:5:
> ../../third_party/openscreen/src/util/crypto/random_bytes.h:12:12: error: use of undeclared identifier 'uint8_t'
> std::array<uint8_t, 16> GenerateRandomBytes16();
>            ^
> ../../third_party/openscreen/src/util/crypto/random_bytes.h:13:26: error: unknown type name 'uint8_t'
> void GenerateRandomBytes(uint8_t* out, int len);
>                          ^

--- third_party/openscreen/src/util/crypto/random_bytes.h.orig	2020-12-10 09:18:10.255337218 -0300
+++ third_party/openscreen/src/util/crypto/random_bytes.h	2020-12-10 09:18:34.219336083 -0300
@@ -6,6 +6,7 @@
 #define UTIL_CRYPTO_RANDOM_BYTES_H_
 
 #include <array>
+#include <stdint.h>
 
 namespace openscreen {
 
